packages/previous/index.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useRef } from 'react'
+
+/**
+ * @param value The value to be returned on the next render.
+ * @returns Returns the last value, undefined for initial render.
+ */
+export default function usePrevious<T>(value: T): T {
```

Comment:
I think you can avoid the parameter type since you already declared it in the `<T>`:
```suggestion
export default function usePrevious<T>(value): T {
```

Reply:
TS sees this as `any` if the type param is not declared.